docs/src/3_minutes.md

```diff
@@ -0,0 +1,64 @@
+```
+
+An `EventSet` can hold one or several time sequences, depending on what its **[index](../user_guide/#index-horizontal-and-vertical-operators)** is.
+
+If the `EventSet` has no index, it will hold a single time sequence, which means that all events will be considered part of the same group and will interact with each other when operators are applied to the `EventSet`.
+
+If the `EventSet` has one (or many) indexes, it will hold one time sequence for each unique value (or unique combination of values) of the indexes, the events will be grouped by their index value, and operators applied to the `EventSet` will be applied to each time sequence independently.
+
+## Graph, `Nodes` and Operators
+
+There are two big phases in any Temporian script: graph **definition** and **evaluation**. This is a common pattern in computing libraries, and it allows us to perform optimizations before the graph is evaluated, share Temporian programs across different platforms, and more.
+
+Processing operations are performed by **operators**. For example, the [`tp.simple_moving_average()`](../reference/temporian/core/operators/window/simple_moving_average) operator computes the [simple moving average](https://en.wikipedia.org/wiki/Moving_average) of each feature in an `EventSet`. You can find documentation for all available operators [here](../reference/temporian/core/operators/all_operators).
+
+Note that when calling operators you are only defining the processing graph - i.e. you are telling Temporian what operations you want to perform on your data, but those operations are not yet being performed.
+
+Operators are not applied directly to `EventSet`s, but to **[Nodes](../reference/temporian/core/data/node)**. You can think of a `Node` as the placeholder for an `EventSet` in the processing graph. When applying an operators to `Node`s, you get back new `Node`s that are placeholders for the results of those operations. You can create arbitrarily complex processing graphs by combining operators and nodes.
+
+```python
+# Obtain the Node corresponding to the EventSet we created above
+source_node = evset.node()
+
+# Apply operators to existing Nodes to generate new Nodes
+sma_node = tp.simple_moving_average(source_node, window_length=tp.duration.hours(12))
+lagged_sma_node = tp.since_last(sma_node, duration=tp.duration.days(7))
+```
+
+<!-- TODO: add image of the generated graph -->
+
+Your preprocessing graph can now be run by calling [`evaluate()`](../reference/temporian/core/data/node/#temporian.core.data.node.Node.evaluate) on any `Node` in the graph, which will perform all necessary operations and return the resulting `EventSet`.
+
+Note that `evaluate()` needs to be passed the `EventSet`s that correspond to the source `Node`s in the graph (since those are not part of the graph definition) and that several `Node`s can be evaluated at the same time by calling [`tp.evaluate()`](../reference/temporian/core/evaluation/#temporian.core.evaluation.evaluate) directly.
```

Comment:
The explanation is not clear to me at least, maybe an alternative is:

Note that `evaluate()` needs to be fed with the input data for the `source` node of the graph, so that the result for this `Node` can be calculated.

Reply:
Also, I'd move this explanation below the code example.